.editorconfig

```diff
@@ -0,0 +1,4 @@
+﻿[*.cs]
+
+# CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
+dotnet_diagnostic.CS8632.severity = none
```

Comment:
For a complete fix, add `#nullable enable` on L853 and `#nullable disable` on L889 of Game1.cs

Reply:
What exactly makes this "improper"? Is there an instance where you wouldn't use annotations? If not, then why not use a .editorconfig setting so we don't have to type out annotations every time we want to have a nullable type?

Reply:
The annotations tell the compiler exactly where nullable variables are expected and legal, the solution you proposed is just suppressing the warning globally.

Reply:
I understand what they are for. What I'm saying is, if there is no situation where this is actually demonstrably useful, then it's useless and we should disable the warning. Matter of fact, why don't we instead remove the ? from the parameter altogether? That also fixes the issue.

Reply:
The nullable aspect is most likely an anti-crash due to unexpected null parameters. Removal of them might break something, but worth investigating. And its not useless, if you read the documentation on nullables, you'd see that its important to prevent null reference exceptions.

https://learn.microsoft.com/en-us/dotnet/csharp/nullable-references
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/nullable-value-types

Reply:
K, I went with removing the ? from the parameters. Microsoft's own examples don't have it. See [here](https://learn.microsoft.com/en-us/dotnet/api/system.windows.forms.control.resize?view=windowsdesktop-8.0&devlangs=csharp&f1url=%3FappId%3DDev16IDEF1%26l%3DEN-US%26k%3Dk(System.Windows.Forms.Control.Resize)%3Bk(DevLang-csharp)%26rd%3Dtrue), [here](https://learn.microsoft.com/en-us/dotnet/api/system.windows.forms.form.resizebegin?view=windowsdesktop-8.0&devlangs=csharp&f1url=%3FappId%3DDev16IDEF1%26l%3DEN-US%26k%3Dk(System.Windows.Forms.Form.ResizeBegin)%3Bk(DevLang-csharp)%26rd%3Dtrue), and [here](https://learn.microsoft.com/en-us/dotnet/api/system.windows.forms.form.resizeend?view=windowsdesktop-8.0&devlangs=csharp&f1url=%3FappId%3DDev16IDEF1%26l%3DEN-US%26k%3Dk(System.Windows.Forms.Form.ResizeEnd)%3Bk(DevLang-csharp)%26rd%3Dtrue).